protocol/contracts/beanstalk/silo/SiloFacet/SiloGettersFacet.sol

```diff
@@ -170,10 +170,10 @@ contract SiloGettersFacet is ReentrancyGuard {
     }
 
     /**
-     * @notice Returns the unclaimed germinating stalk for a season.
+     * @notice Returns the unclaimed germinating stalk and roots for a season.
      */
-    function getGerminatingStalkForSeason(uint32 season) external view returns (uint256) {
```

Comment:
Should we leave `getGerminatingStalkForSeason` and `getGerminatinRootsForSeason` in and also add `getGerminatingStalkAndRootsForSeason`?

Reply:
https://github.com/BeanstalkFarms/Beanstalk/commit/f77a9b59c88a8cf92a3a3bd72feafe4fd580d244